src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcTokenHandlerFactory.php

```diff
@@ -90,6 +90,8 @@ public function addConfiguration(NodeBuilder $node): void
         $node
             ->arrayNode($this->getKey())
                 ->fixXmlConfig($this->getKey())
+                ->fixXmlConfig('issuer')
```

Comment:
This one is for 7.3.

Reply:
See 
* https://github.com/symfony/symfony/pull/61701